03_sum_numbers.rb

```diff
@@ -0,0 +1,22 @@
+# Sum Numbers
+#
+# Write a method that will take an array of numbers,
+# and return their sum.
+#
+# Difficulty:
+# 3/10
+#
+# Example:
+# sum_numbers([1,1,1]) should return 3
+# sum_numbers([5,2,100,0,10]) should return 117
+#
+# Check your solution by running the tests:
+# ruby tests/03_sum_numbers_test.rb
+#
+
+def sum_numbers (numbers)
+  numbers.inject :+
```

Comment:
Didn't know you could do this. Nice solution.

Reply:
neither did I